page-objects/pages/DashboardPage.js

```diff
@@ -13,8 +13,8 @@ const taskElement = {
 };
 
 const taskSection = {
-	assigned: '#main-content > section:nth-child(3)',
-	private: '#main-content > section:nth-child(4)',
+	assigned: 'section[data-testid="published-tasks-section"]',
```

Comment:
```suggestion
	published: 'section[data-testid="published-tasks-section"]',
```
one naming would be better :)

Reply:
done